qmlti/lib.php

```diff
@@ -157,12 +157,18 @@ function init_db($db) {
 
         $sql = <<< EOD
 CREATE TABLE [dbo].[{$consumer_table_name}] (
-  [consumer_key] VARCHAR(50) NOT NULL,
+  [name] VARCHAR(50) NOT NULL,
   [secret] VARCHAR(50) NOT NULL,
-  [consumer_name] VARCHAR(20) NOT NULL,
-  [customer_id] VARCHAR(25) NOT NULL,
-  [username_prefix] VARCHAR(10) NULL,
-  [last_access] DATE NULL,
+  [lti_version] VARCHAR(20) NOT NULL,
+  [consumer_name] VARCHAR(25) NOT NULL,
+  [consumer_version] VARCHAR(10),
+  [consumer_guid] VARCHAR(25),
+  [css_path] VARCHAR(25),
+  [protected] INT,
+  [enabled] INT,
+  [enable_from] DATETIME,
+  [enable_until] DATETIME,
+  [last_access] DATETIME NULL,
   [created] DATETIME NOT NULL,
   [updated] DATETIME NOT NULL,
  CONSTRAINT [PK_{$consumer_table_name}] PRIMARY KEY CLUSTERED ([consumer_key] ASC)
```

Comment:
consumer_key is still the PK (which probably makes sense on a table called consumer)